docs/portal/image-registry/viewing-logs.md

```diff
@@ -1,10 +1,13 @@
 # Viewing Logs
 
-ECP keeps a log of your registry projects. The log contains a date- and time-stamped record of the operations performed by users. The following procedure describes how to view logs. To make sure you view the logs for the appropriate project, you can view a project's information before viewing its logs.
+ECP keeps a log of your registry projects. The log contains a date- and time-stamped record of the operations performed by registry users. The following procedure describes how to view logs. To make sure you view the logs for the appropriate project, you can view a project's information before viewing its logs.
+
+**Note** Users with project update permission can edit this page. Users without project update permission can only read this page.
```

Comment:
The phrase "this page" does not look like clear enough here. Are you referring to the logs list or the project details page? If it is the former one, note that the logs are readonly themselves. You can't edit logs. 

I'd suggest this sentence be simply removed. The focus on this page is logs, and since there is no specific access control imposed againts viewing logs, a caveat is not necessary.

Reply:
Removed. This might have been a copy and paste error.